pkg/yang/types.go

```diff
@@ -23,6 +23,12 @@ import (
 	"sync"
 )
 
+// UsePosixPatternExt specifies that openconfig-extensions:posix-pattern should
```

Comment:
Yes that's right... the only issue might be that the users' custom YANG files would need to be converted as well. However, I'm leaning towards not providing a transition exception file list because it just makes the tools more complex in having to support two different ways to parse files. I'd rather not have to deal with the correctness of that (e.g. augments, deviations, submodules, etc.)

Reply:
Yes that's right... I'm leaning towards not providing a transition exception file list because it just makes the tools more complex in having to support two different ways to parse files. Dealing with the correctness of that sounds complex (e.g. augments, deviations, submodules, etc.).